torch/csrc/jit/codegen/cuda/kernel_cache.h

```diff
@@ -19,14 +19,34 @@ namespace cuda {
 // to the instance of `InputsIdLookup`.
 class InputsIdLookup {
```

Comment:
should this be TORCH_CUDA_API ?

Reply:
I only exposed the `lookupId`, I'll move that up here.